app/controllers/concerns/idv/document_capture_concern.rb

```diff
@@ -0,0 +1,21 @@
+module Idv
+  module DocumentCaptureConcern
+    extend ActiveSupport::Concern
+
+    included do
+      before_action :override_document_capture_step_csp
+    end
+
+    def override_document_capture_step_csp
```

Comment:
So for me, my gut would be to make this a "plain" mixin, ie just a module with methods, and then leave the `before_action :override_document_capture_step_csp`  line in the controllers. It helps with discoverability, because then we can, in the controller, see all the things that get run.

```suggestion
    def override_document_capture_step_csp
```

Reply:
Seems reasonable to me 👍  I agree the explicit mixin use is much easier to follow, and only marginally more verbose (one extra line per controller).